definitions/marts/looker_studio_marts/ecf/ls_ecf_provider_activity_partnership_creations.sqlx

```diff
@@ -0,0 +1,63 @@
+config {
+    type: "table",
+    bigquery: {
+        partitionBy: "event_date",
+        clusterBy: ["cpd_lead_provider_name"]
+    },
+    description: "This mart contains successful (200 status code) API requests made by a Lead Provider on the API path containing 'partnerships/ecf' to create a partnership between them, a school and a delivery partner. The events captured in this mart equate to the majority in relation to creating partnerships; there are some providers who still use an alternative method of providing a csv for uploading which this mart does not contain. \n \

```

Comment:
"The events captured in this mart equate to the majority in relation to creating partnerships" - Maybe: This mart captures the majority of created partnerships.